presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcDeleteDeltaPageSource.java

```diff
@@ -104,14 +102,15 @@ public OrcDeleteDeltaPageSource(
             Map<String, OrcColumn> acidColumns = uniqueIndex(
                     reader.getRootColumn().getNestedColumns(),
                     orcColumn -> orcColumn.getColumnName().toLowerCase(ENGLISH));
-            List<OrcColumn> rowIdColumns = ImmutableList.of(
+
+            List<OrcColumn> rowIdColumns;
+            rowIdColumns = ImmutableList.of(
                     acidColumns.get(ACID_COLUMN_ORIGINAL_TRANSACTION.toLowerCase(ENGLISH)),
-                    acidColumns.get(ACID_COLUMN_BUCKET.toLowerCase(ENGLISH)),
                     acidColumns.get(ACID_COLUMN_ROW_ID.toLowerCase(ENGLISH)));
 
             recordReader = reader.createRecordReader(
                     rowIdColumns,
-                    ImmutableList.of(BIGINT, INTEGER, BIGINT),
```

Comment:
as above

Reply:
as explained above, to avoid reading the bucket id column.